packages/fxa-support-panel/test/lib/api.spec.ts

```diff
@@ -123,15 +123,25 @@ describe('Support Controller', () => {
     cassert.equal(result.statusCode, 200);
   });
 
-  it('gracefully handles 404s', async () => {
+  it('gracefully handles 404s/500', async () => {
     const defaults = createDefaults();
     defaults.account.status = 404;
     mockCalls(defaults);
-    const result = await server.inject({
+    let result = await server.inject({
+      method: 'GET',
+      url: `/?uid=${uid}`
+    });
+    cassert.equal(result.statusCode, 500);
```

Comment:
I think this should be `404`.

Reply:
I had considered that originally, but if its unable to fetch URL's that should work with expected status codes, there's an error occurring. It logs an error, and it seems reasonable to indicate a service error with a 500 as well.